addons/website/static/src/snippets/s_mega_menu_odoo_menu/000.scss

```diff
@@ -2,7 +2,10 @@
 .s_mega_menu_odoo_menu:not([data-vcss]) {
     .s_mega_menu_odoo_menu_footer {
         // Apply color transparency to match with the preset used
-        border-color: rgba(0, 0, 0, .05) !important;
-        background: rgba(0, 0, 0, .05) !important;
+        border-color: rgba(0, 0, 0, .05);
+
+        .row > div {
+            background-color: rgba(0, 0, 0, .05);
+        }
```

Comment:
This one is also a stable style change... but in this case I judge it's ok as it fixes more problems that the potential "break"